src/test/java/uk/gov/hmcts/reform/migration/service/RespondentsDataMigrationServiceTest.java

```diff
@@ -0,0 +1,303 @@
+        assertThat(firstRespondentParty.get("email")).isNull();
+        assertThat(firstRespondentParty.get("gender")).isNull();
+        assertThat(firstRespondentParty.get("genderIdentification")).isNull();
+        assertThat(firstRespondentParty.get("placeOfBirth")).isNull();
+        assertThat(firstRespondentParty.get("relationshipToChild")).isNull();
+        assertThat(firstRespondentParty.get("contactDetailsHidden")).isNull();
+        assertThat(firstRespondentParty.get("litigationIssues")).isNull();
+        assertThat(firstRespondentParty.get("litigationIssuesDetails")).isNull();
+    }
+
+    @Test
+    public void whenOldRespondentWithOneNameIsMigratedShouldReturnNewListStructureWithOnlyFirstName() {
+        Map<String, Object> data = new HashMap<>();
+
+        OldRespondent respondent = OldRespondent.builder()
+            .name("Beyoncé")
+            .build();
+
+        data.put("respondents", OldRespondents.builder()
+            .firstRespondent(respondent)
+            .build());
+
+        CaseDetails caseDetails = CaseDetails.builder()
+            .id(1111L)
+            .data(data)
+            .build();
+
+        service.migrate(caseDetails);
+
+        List<Map<String, Object>> respondents = (List<Map<String, Object>>) caseDetails.getData().get("respondents1");
+
+        Map<String, Object> firstRespondent = (Map<String, Object>) respondents.get(0).get("value");
+        Map<String, Object> firstRespondentParty = (Map<String, Object>) firstRespondent.get("party");
+
+        assertThat(respondents).hasSize(1);
+        assertThat(firstRespondentParty.get("firstName")).isEqualTo("Beyoncé");
+        assertThat(firstRespondentParty.get("lastName")).isNull();
+    }
+
+    @Test
+    public void whenOldRespondentWithManyNamesIsMigratedShouldReturnNewListStructureWithMultipleFirstNames() {
+        Map<String, Object> data = new HashMap<>();
+
+        OldRespondent respondent = OldRespondent.builder()
+            .name("Jean Paul Gautier")
+            .build();
+
+        data.put("respondents", OldRespondents.builder()
+            .firstRespondent(respondent)
+            .build());
+
+        CaseDetails caseDetails = CaseDetails.builder()
+            .id(1111L)
+            .data(data)
+            .build();
+
+        service.migrate(caseDetails);
+
+        List<Map<String, Object>> respondents = (List<Map<String, Object>>) caseDetails.getData().get("respondents1");
+
+        Map<String, Object> firstRespondent = (Map<String, Object>) respondents.get(0).get("value");
+        Map<String, Object> firstRespondentParty = (Map<String, Object>) firstRespondent.get("party");
+
+        assertThat(respondents).hasSize(1);
+        assertThat(firstRespondentParty.get("firstName")).isEqualTo("Jean Paul");
+        assertThat(firstRespondentParty.get("lastName")).isEqualTo("Gautier");
+    }
+
+    private Map<String, Object> createOldRespondent(boolean multiple, boolean partial) {
```

Comment:
There is not test that would simulate existence of additional respondents as `multiple` flag is always set false.